include/laser_filters/intensity_filter.h

```diff
@@ -88,12 +88,13 @@ class LaserScanIntensityFilter : public filters::FilterBase<sensor_msgs::LaserSc
     {
       if (filtered_scan.intensities[i] <= lower_threshold_ ||                           // Is this reading below our lower threshold?
           filtered_scan.intensities[i] >= upper_threshold_)                             // Is this reading above our upper threshold?
-      {      
-
+      {
```

Comment:
No, but I think it is strange to remain this empty line, so I deleted.
I will remove this fix.